CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest
+- Changed default resolution of Camera2VideoSourceImpl to 960*540
+- Added optional parameter to Camera2VideoSourceImpl constructor to enable\disable autofocus
+- Added support of reverse landscape orientation
+- Removed deprecated code for 0.12.0:
+  - `VisionManager.setModelPerformanceConfig`
+  - `VisionReplayManager.setModelPerformanceConfig`
+  - `SystemInfoUtils.getSnpeSupportedBoard`
+  - `SystemInfoUtils.getSystemProperty`
+  - `enum class SupportedSnapdragonBoards`
+  - `class ModelPerformanceConfig`
+- Integrated new rendering engine for VisionView and VisionArView with better performance
+- Integrated LaneDetection mode for VisionView
+- Deprecated `VisionArEventsListener.onArCameraUpdated` method
+- Extended `Camera` class returned by `VisionEventsListener.onCameraUpdated`
+- Added Okinawa to Japan sign classifier
+- Improved segmentation rendering
```

Comment:
What are the improvements?

Reply:
https://github.com/mapbox/mapbox-vision/pull/2554
Probably it doesn't deserve description

Reply:
I wouldn't list them then

Reply:
removed